### ISSUE TITLE
Shortcodes not being expanded in saved blocks

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik#129]
In wp-pompey.org.uk I have a paragraph that includes the [bw_link] shortcode to two internal pages. I saved the block as a reusable block and used it in another post. On the front end of the new post the shortcodes have not been expanded.
```
Thank you to our group sponsors [bw_link 833] and 
[bw_link 1002] for supporting us with a venue and refreshments.
```
### Expected output
Shortcodes to be expanded in the reusable block. 

### Workaround
Change the shortcodes to standard WordPress links.

**Comment by bobbingwide** [comment]
This needs to be investigated. Find out if the proposed fix to #120 is affecting the results.

**Comment by bobbingwide** [comment]
### Problem explanation
> This could be because `oik_add_shortcodes` has been run.

This was the case.  `oik_do_shortcode` does not notice the shortcodes in the reusable blocks since the post content just contains, as an example, `<!-- wp:block {"ref":20793} /-->`. 

A simple fix is to change the test to check that there aren't any reusable blocks in the content. 
A better fix might be to change the priority of the filter function against `the_content`.

**Comment by bobbingwide** [comment]
This was fixed in v3.3.6, but only for the version delivered from oik-plugins.com. It’s now in v4.0.0 on wordpress.org, so closing.
